awsService/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.13 as builder
+WORKDIR /go/src/github.com/codeuniversity/smag-mvp
+COPY . .
```

use the fixed build-cache, to make the build *way* faster:
```suggestion
COPY go.mod .
COPY go.sum .
RUN GO111MODULE=on go mod download
COPY . .
```